debezium-connector-mongodb/src/test/java/io/debezium/connector/mongodb/transforms/ExtractNewDocumentStateTestIT.java

```diff
@@ -1238,7 +1238,7 @@ public void shouldPropagatePreviousRecordHeaders() throws InterruptedException {
         // Perform transformation
         final SourceRecord transformed = transformation.apply(record);
 
-        assertThat(transformed.headers()).hasSize(1);
+        assertThat(transformed.headers()).hasSize(4);
```

Comment:
I think for future refernce in this and other places the count of headers should not be hardcoded but conturcted dynamically - there should be a method to which for example 1 is added and it returns 1 + Debezium standard headers.